src/test/integration/util/common.ts

```diff
@@ -102,7 +102,7 @@
 	public async getErrorOutput (): Promise<string> {
```

Comment:
<!--
  0 failure: 
  1 warning:  src/test/integrat...
  
  
  DangerID: danger-id-Danger;
  File: src/test/integration/util/common.ts;
  Line: 68;
-->

- :warning: src/test/integration/util/common.ts line 68 – Argument 'tiapp' should be typed with a non-any type. (@typescript-eslint/explicit-module-boundary-types)